operators/admin/olm-managing-po.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="olm-managing-po"]
+= Managing platform Operators (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olm-managing-po
+
+toc::[]
+
+A platform Operator is an OLM-based Operator that can be installed during or after an OpenShift Container Platform cluster's Day 0 operations and participates in the cluster's lifecycle. As a cluster administrator, you can manage platform Operators by using the `PlatformOperator` API.
+
+:FeatureName: The platform Operator type
+include::snippets/technology-preview.adoc[]
+
+include::modules/arch-platform-operators.adoc[leveloffset=+1]
+
+[id="prerequisites_olm-managing-po"]
+== Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- The `TechPreviewNoUpgrades` feature set enabled on the cluster.
+- Only the `redhat-operators` catalog source enabled on the cluster. This is a restriction during the Technology Preview release.
```

Comment:
I hadn't seen this mentioned anywhere until I tried to install with the default catalog sources enabled, and got these errors until I disabled everything but `redhat-operators`:

```
message: 'validation error: only a single catalog source is supported during phase
 0: failed to run sourcing logic'
```

```
    message: 'CatalogSource.operators.coreos.com "redhat-operators" not found: failed
      to run sourcing logic'
    reason: SourceFailed
    status: "False"
```